tasmota/xdrv_60_shift595.ino

```diff
@@ -0,0 +1,114 @@
+/*
+  xdrv_60_shift595.ino - 74x595 shift register family support for Tasmota
+
+  Copyright (C) 2021  Jacek Ziółkowski
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#ifdef USE_SHIFT595
+
+#define XDRV_60           60
+
+const char kShift595Commands[] PROGMEM = "|" D_CMND_SHIFT595_DEVICE_COUNT ;
+void (* const Shift595Command[])(void) PROGMEM = { &CmndShift595Devices };
```

Comment:
If you use `Shift595` as a prefix, that will open the door to more commands :
```
const char kShift595Commands[] PROGMEM = D_PRFX_SHIFT595 "|" D_CMND_SHIFT595_DEVICE_COUNT ;
```
With
```
#define D_PRFX_SHIFT595 "Shift595"
#define D_CMND_SHIFT595_DEVICE_COUNT "Devices"
```

Reply:
yes, I did consider that but with one command currently it's pointless. Can be always added.